### PR TITLE
update gazelle to 0.5.3

### PR DESCRIPTION
**Pull request by daiki44**
# What?
- update subscribe sync event

### CODE REVIEW

File: src/store/appStatus.js [hunk 1]
```diff
@@ -193,34 +194,46 @@ export const initializeMagicLinkWallet = email => {
...
   return (dispatch, getState) => {
-    client.subscribeCheckpointFinalized((checkpointId, checkpoint) => {
+    client.subscribeCheckpointFinalized(async (checkpointId, checkpoint) => {
       console.info(
         `new %ccheckpoint %cdetected: %c{ id: ${checkpointId.toHexString()}, checkpoint: (${checkpoint}) }`,
         'color: pink; font-weight: bold;',
         '',
         'font-weight: bold;'
       )
+      await sleep(500)
```
Comment (syuhei176) [inline src/store/appStatus.js:204]: is it needed?
I guess subscribedEventGetters call getters after syncingStatus is LOADED.
and subscribeCheckpointFinalized doesn't support async handler function.

Comment (daiki44) [reply src/store/appStatus.js:204]: Sometimes, we can't get the balance immediately after deposit.
I guess the reason is getting balance before syncing.
That's why I added it.

What do you think?

Comment (daiki44) [reply src/store/appStatus.js]: Removed!

But we have to resolve the problem that sometimes it can't get balance immediately after deposit.

Comment (syuhei176) [reply src/store/appStatus.js]: I got it. I think we can modify gazelle like calling handler after balance updated!
https://github.com/cryptoeconomicslab/gazelle/blob/master/packages/plasma-light-client/src/LightClient.ts#L342

**Pull request merged**